### PR TITLE
Paragraph Field Descriptions

### LINKED ISSUE #179: Paragraph field descriptions
A goal for D8 is to document as much as possible within the site. Field description should be provided for each field.

Provide field descriptions for paragraphs fields.
- Admin title
- Background image
- Content Width
- Content Vertical Alignment
- Unique ID

### PR DESCRIPTION
**Pull request by joewhitsitt**
resolves #179

- also checks for unique id value before processing in id attribute.

### CODE REVIEW

**Comment by richardbporter** [comment]
Resolved conflict based on https://github.com/uiowa/rusty/commit/986b11a5c0a5e084d5179d4fda8e30d527033197#diff-27cd2c95050668dfd9d721733ce00eb2.

File: modules/uiowa_paragraphs/config/install/field.field.paragraph.section.field_section_content_block.yml [hunk 1]
```diff
@@ -23,8 +23,6 @@ settings:
     negate: 0
     target_bundles:
       text: text
-      accordion: accordion
-      card: card
```
Comment (richardbporter) [inline modules/uiowa_paragraphs/config/install/field.field.paragraph.section.field_section_content_block.yml]: What is this change doing?

Comment (joewhitsitt) [reply modules/uiowa_paragraphs/config/install/field.field.paragraph.section.field_section_content_block.yml]: something got messed up. default install should have card and accordion as bundle options for the section paragraph reference field.

**Comment by joewhitsitt** [comment]
I am going to look into the target bundles issue, but I think this PR is blocked due to possible label changes that were talked about in #197 . @pyrello were you planning on changing field names/machine_names for the paragraph fields?

**Comment by richardbporter** [comment]
These are better than no descriptions given the current field names. #197 can update the descriptions as needed.

**Pull request merged**